plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/animator/CameraAnimator.kt

```diff
@@ -10,6 +11,7 @@ import com.mapbox.maps.plugin.animation.CameraAnimatorType
 /**
  * Base generic class for all camera animators.
  */
+@SuppressLint("Recycle")
```

Comment:
As a nit here and in other place below - can we mark it not for the whole class but for specific method?

Reply:
Since lint error is related to `ValueAnimator` and `CameraAnimator` class extends that.`supressLint` needs to be at the class level.